docs/about.md

```diff
@@ -24,22 +24,27 @@ back re-encryption / decryption of data onchain.
 Imagine Alice wants to sell a picture of her cat for 1 DAI on an eBay-like dApp. 
 
 ### 1. Uploading
+
 Alice encrypts the cat picture to the Medusa network and registers the encryption and the price to the dApp platform.
 
 ### 2. Buying
 
-When Bob wants the buy the cat picture, it sends one 1 DAI to Alice via the dApp 
-platform. At this point, the dApp asks Medusa (on smart contract) to _reencrypt_ the cat picture to Bob.
+When Bob wants the buy the cat picture, he sends one 1 DAI to Alice via the dApp
+platform. At this point, the dApp smart contract requests Medusa to _reencrypt_ the cat picture to Bob.
 
 ### 3. Reencryption to Bob
 
 Medusa verifies that the cat was submitted to the same platform that requested the
-reencryption. If so, Medusa's nodes perform their operation offchain and submits the
+reencryption. If so, Medusa nodes perform their operation offchain and submit the
 resulting reencryption onchain.
 
 ### 4. Local decryption
 
-Bob sees the event that the reencryption has been submitted, and can now locally decrypts
-the cat picture !
+Bob sees the event that the reencryption has been submitted, and can now locally decrypt
+the cat picture!
+
+:::info Important
+
+Only Bob can decrypt the cat picture. Medusa's [threshold encryption protocol](./about/cryptography/threshold-network.md) ensures the dApp and Medusa nodes **cannot** decrypt and see the cat picture.
 
-**Important**: At no point in time, the dApp nor Medusa nodes were able to decrypt and see the cat picture.
```

Comment:
You dont think it's important to mention that ? @jaeaster

Reply:
It's still there. I reworded it and put it in a highlighted section:

```
:::info Important

Only Bob can decrypt the cat picture. Medusa's [threshold encryption protocol](./about/cryptography/threshold-network.md) ensures the dApp and Medusa nodes **cannot** decrypt and see the cat picture.

:::
```